src/features/orientation/infrastructure/presentation/layouts/filter/filter.presenter.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, of } from 'rxjs';
+import { GeolocationPresenter, LieuxMediationNumeriqueListPresenter } from '../../../../../cartographie/domain';
+import { LieuMediationNumerique } from '../../../../../../models/lieu-mediation-numerique/lieu-mediation-numerique';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FilterPresenter {
+  public filter$: BehaviorSubject<{ name: string; type: string }[]> = new BehaviorSubject<{ name: string; type: string }[]>([]);
```

Comment:
```suggestion
  private _filters$: BehaviorSubject<{ name: string; type: string }[]> = new BehaviorSubject<{ name: string; type: string }[]>([]);
  public filters$: Observable<{ name: string; type: string }[]> = this._filters.asObservable();
```
Créer un type pour `{ name: string; type: string }`, `Filter` par exemple ?